### PR TITLE
SDK updates according to new API

### PR DESCRIPTION
**Pull request by NikitaSkrynnik**
## Description
SDK updates according to new [API](https://github.com/networkservicemesh/api/pull/114)

## Issue link
Closes https://github.com/networkservicemesh/sdk/pull/1131

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [ ] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [ ] New functionallity
- [ ] Documentation
- [x] Refactoring
- [ ] CI

### CODE REVIEW

File: pkg/registry/common/localbypass/server_test.go [hunk 1]
```diff
@@ -123,9 +123,9 @@ func TestLocalBypassNSEServer_Restart(t *testing.T) {
 	})
 	require.NoError(t, err)
 
-	findNSE, err := stream.Recv()
+	findNSER, err := stream.Recv()
```
Comment (denis-tingaikin) [inline pkg/registry/common/localbypass/server_test.go:126]:
```suggestion
	nseStream, err := stream.Recv()
```

File: pkg/registry/common/memory/ns_server_test.go [hunk 2]
```diff
@@ -194,7 +199,7 @@ func TestNetworkServiceRegistryServer_SlowReceiver(t *testing.T) {
 
 	ignoreCurrent := goleak.IgnoreCurrent()
 
-	_, err := receiveNS(findCtx, ch)
+	_, err := receiveNSR(findCtx, ch)
```
Comment (denis-tingaikin) [inline pkg/registry/common/memory/ns_server_test.go:202]: [minior]
```suggestion
	_, err := readNSResponse(findCtx, ch)
```

**Review by denis-tingaikin — COMMENTED** [review]
Added a few minor comments. In general, look good.

File: pkg/registry/chains/memory/server_test.go [hunk 3]
```diff
@@ -61,7 +61,7 @@ func Test_RegistryMemory_ShouldSetDefaultPayload(t *testing.T) {
 	nsrfc, err := nsrc.Find(ctx, &registry.NetworkServiceQuery{NetworkService: ns})
 	require.NoError(t, err)
 
-	ns, err = nsrfc.Recv()
+	_, err = nsrfc.Recv()
```
Comment (denis-tingaikin) [inline pkg/registry/chains/memory/server_test.go:64]: Why is ns removed?

**Review by denis-tingaikin — APPROVED** [review]

File: pkg/registry/common/localbypass/server_test.go [hunk 4]
```diff
@@ -69,19 +69,19 @@ func TestLocalBypassNSEServer(t *testing.T) {
 	})
 	require.NoError(t, err)
 
-	findNSE, err := stream.Recv()
+	nseStream, err := stream.Recv()
```
Comment (Bolodya1997) [inline pkg/registry/common/localbypass/server_test.go:72]: It is not a stream, it is a response :)

Comment (Bolodya1997) [reply pkg/registry/common/localbypass/server_test.go:72]:
```suggestion
	nseResp, err := stream.Recv()
```

**Review by Mixaster995 — APPROVED** [review]

**Review by Bolodya1997 — APPROVED** [review]

**Review by glazychev-art — APPROVED** [review]

**Pull request merged**